docs/cookbook/usage_tips.rst

```diff
@@ -2,7 +2,7 @@
 API Usage Tips
 ##############
 
-Below is a list of helpful tips when using the Flow Production Tracking API. We have tried to make the API very
+Below is a list of helpful tips when using the FPTR API3. We have tried to make the API very
```

Comment:
You need to use the long version because the short version has not being defined in this document yet

```suggestion
Below is a list of helpful tips when using the Flow Production Tracking API3. We have tried to make the API very
```